### ISSUE TITLE
Undefined index captcha in catalog/controller/extension/captcha/basic_captcha.php

### ISSUE THREAD

**Issue by thesamarkand** [opencart/opencart#5140]
```
PHP Notice: Undefined index: captcha in /catalog/controller/extension/captcha/basic_captcha.php on line 24
23: 
24:  if (empty($this->session->data['captcha']) || ($this->session->data['captcha'] != $this->request->post['captcha'])) {
25:  return $this->language->get('error_captcha');

```

Fix:

`
if (empty($this->session->data['captcha']) || empty($this->request->post['captcha']) || ($this->session->data['captcha'] != $this->request->post['captcha'])) {
`

**Comment by arnisjuraga** [comment]
Can you post a pull request?

**Comment by pnair39** [comment]
@arnisjuraga   I was not able to re-create the issue on opencart 2.3.0.2.  What version of opencart is this related too?

**Comment by thesamarkand** [comment]
Version 2.3.0.2 (trs-2.3.0.0)

С уважением, Samarkand.

  02.03.2017, 5:00:16 пользователь pnair39 (notifications@github.com) написал:


  @arnisjuraga I was not able to re-create the issue on opencart 2.3.0.2. What
  version of opencart is this related too?

**Comment by danielkerr** [comment]
fixed in 3.0.0
